src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+
+export function list(settings, logger) {
+  const files = fs.readdirSync(settings.pluginDir);
+  files
+  .filter(function (file) {
+    return file[0] !== '.';
```

Comment:
Actually, I think I can remove this filter altogether. It was meant to check if the first character of a filename is `.` because in the `installedPlugins` source directory there is a `.empty` file (presumably so the directory is preserved in git). From what I can tell this `.empty` file does not actually get deployed as part of Kibana.